docs/source/features.rst

```diff
@@ -46,7 +70,7 @@ showing a compilation bottleneck.
 Limitations
 ===========
 
-Dependency detection
+Single line IF calls
 --------------------
 Whilst fab can automatically determine dependencies from module use statements,
```

Comment:
Assuming Fortran again. This is not relevant to C, I presume. Further more it duplicates a lot of information with the "Advanced config" page. Probably best to keep this page clear of technical details and just point to the work around.

Reply:
Good call.